src/interfaces/store-interfaces/state-model-interface.ts

```diff
@@ -38,5 +38,6 @@ export interface StateModelIF {
   validationFlags: ValidationFlagsIF
   summaryMode: boolean,
   currentFees: FeesIF,
-  feePrices: FeesIF
+  feePrices: FeesIF,
+  createResolutionStep: CreateResolutionIF
```

Comment:
Let's get away from calling these "steps" (which is left over from Create UI).

This is just a data object, and could be reused by any filing.